docs/semgrep-supply-chain/triage-remediation.md

```diff
@@ -102,49 +55,80 @@ Use filters to narrow down your results. The following criteria are available fo
 
-#### Status
+### Status
 
 The triage state of the finding:
 
 * **Open**: Findings for which there have been no triage or remediation action.
 * **Reviewing**: Findings that require more investigation to determine what the next steps should be.
 * **Fixing**: Findings for which you have decided to fix. Commonly used to indicate that these findings are tracked in Jira or assigned to developers for further work.
-* **Ignored**: Vulnerabilities that have been triaged as **Ignored** by the user. You can filter findings with a status of **Ignored** further by reason:  **False positive**, **Acceptable risk**, **No time to fix**, or **No triage reason**.
+* **Ignored**: Vulnerabilities that have been triaged as **Ignored** by the user. You can filter findings with a status of **Ignored** further by reason: **False positive**, **Acceptable risk**, **No time to fix**, or **No triage reason**.
 * **Fixed**: Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as **Fixed**.
 
 > You can set the **Fixing** and **Reviewing** statuses only if you are a [Jira beta](https://semgrep.dev/docs/semgrep-appsec-platform/jira) participant.
 
-#### Transitivity
+### Transitivity
 
 The transitivity of the finding: 
 
 * **Direct**: Your project depends directly on the dependency.
 * **Transitive**: Your project's dependency depends on a vulnerable dependency.
 * **Undetermined**: Semgrep had no transitivity information for the dependency as it relates to your project.
 
-#### EPSS probability
+### EPSS probability
 
 The [Exploit prediction scoring system (EPSS) probability](https://www.first.org/epss/) represents the likelihood that the vulnerability will be exploited in the wild in the next 30 days. Its values range from 0% to 100%. The higher the score, the greater the probability the vulnerability will be exploited. Semgrep groups probabilities as follows:
 
 * <b>High</b>: 50 - 100%
 * <b>Medium</b>: 10 - &#60;50%
 * <b>Low</b>: &#60;10%
 
-#### Reachability
+### Reachability
 
-The finding's exposure to potential attacks, or whether it is reachable or not.
+The finding's exposure to potential attacks, or whether it is reachable.
 
 * **Reachable**: A finding is reachable if there's a code pattern in the codebase that matches the vulnerability definition.
 * **Always reachable**: A finding is always reachable if it's something Semgrep recommends fixing, regardless of what's in the code.
 * **Conditionally reachable**: A finding is conditionally reachable if Semgrep finds a way to reach it when scanning your code when certain conditions are met.
-* **No Reachability Analysis**: A finding that isn't scanned by Semgrep for reachability.
+* **No Reachability Analysis**: A finding that Semgrep doesn't scan for reachability.
```

Comment:
I just finished the `supported-languages.md` review today and I noticed that we list this as **Unknown** - maybe worth it to update the `supported-languages.md` in this PR as well to be consistent with this?